.github/workflows/upstream.yml

```
@@ -73,7 +73,7 @@ jobs:
           && github.event_name != 'pull_request'
           && github.repository == 'dask/dask'
           && steps.run_tests.outcome == 'failure'
-        uses: xarray-contrib/issue-from-pytest-log@v1.2
+        uses: xarray-contrib/issue-from-pytest-log@v1.2.3
```

```suggestion
        uses: xarray-contrib/issue-from-pytest-log@v1.2.4
```
I just released a new version to also create an issue if the test collection step fails.